packages/ui/src/components/alert-dialog.tsx

```diff
@@ -26,7 +26,7 @@ const AlertDialogOverlay = React.forwardRef<
     <AlertDialogPrimitive.Overlay
       className={cn(
         'data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 dialog-backdrop fixed inset-0 z-50',
-        { 'bg-cn-background-backdrop': isLightTheme },
+        // { 'bg-cn-background-backdrop': isLightTheme },
```

Comment:
This can be removed, `dialog-backdrop` class will handle this based on the theme.